Makefile-docker

```diff
@@ -135,7 +97,7 @@ update_assets:
 	$(PYTHON_COMMAND) manage.py collectstatic --noinput
 
 .PHONY: update
-update: update_deps update_db update_assets ## update the dependencies, the database, and assets
+update: update_db update_assets ## update the dependencies, the database, and assets
```

Comment:
Something I missed in the changes in https://github.com/mozilla/addons-server/commit/bc186195bc5da7fc0f7268a11f67a5470a64c454 was that `update_docker` (in Makefile-os) no longer calls this `update` recipe, so `update_db` doesn't get called anywhere automatically. Maybe `update_assets` should be executed with `up` and this recipe just removed, leaving `make update_db` to be stand-alone command.

I think calling `update_deps` too as part of `up` is risky though - if there are any db conflicts (which can happen often if you've been making changes in one of your patches) there wouldn't be an easy way to recover.

Reply:
It is impossible to call `update_deps` at all on a running container. The thing is.. `update_assets` and `update_deps` are both already called during `up` because, well you build an image and in it are the deps and the compiled/collected assets.

I have a separate PR (this one is split from) to run db updates as a part of up. It was getting to big and this part seemed relatively straight forward.

in the end I don't think there will need to be any commands starting with "update" as they all just get rolled into "up"

Reply:
gah, I meant to write:

> I think calling **update_db** too as part of `up` is risky though

Migrations can easily fail if you've been making database changes yourself and `make up` can't fail or we can't get into the container to investigate/fix the problem